hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ReconTaskControllerImpl.java

```diff
@@ -179,8 +186,8 @@ public void reInitializeTasks(OMMetadataManager omMetadataManager)
       List<Future<Pair>> results = executorService.invokeAll(tasks);
       for (Future<Pair> f : results) {
         String taskName = f.get().getLeft().toString();
-        if (!(Boolean)f.get().getRight()) {
-          LOG.info("Init failed for task : " + taskName);
+        if (!(boolean)f.get().getRight()) {
```

Comment:
Although Sonar did not point it out: simply replacing `Pair` with `Pair<String, Boolean>` would allow getting rid of the unsafe casts completely.

Reply:
Thank you @adoroszlai. Done